larq_compute_engine/core/bgemm_impl_ruy.h

```diff
@@ -38,17 +39,13 @@ struct BGemmImplUsingRuy {
 
     // Set up the matrix layouts and mul_params.
     ruy::Matrix<TBitpacked> lhs;
-    ruy::MakeSimpleLayout(lhs_params.rows, lhs_params.cols,
-                          ruy::Order::kRowMajor, lhs.mutable_layout());
     ruy::Matrix<TBitpacked> rhs;
-    ruy::MakeSimpleLayout(rhs_params.rows, rhs_params.cols,
-                          ruy::Order::kColMajor, rhs.mutable_layout());
     ruy::Matrix<DstScalar> dst;
-    ruy::MakeSimpleLayout(dst_params.rows, dst_params.cols,
-                          ruy::Order::kColMajor, dst.mutable_layout());
-    lhs.set_data(lhs_data);
-    rhs.set_data(rhs_data);
-    dst.set_data(dst_data);
+    cpu_backend_gemm::detail::MakeRuyMatrix(lhs_params, lhs_data, &lhs,
+                                            context->use_caching());
```

Comment:
What about hard coding `use_caching = true` instead of infering the value from the context which is currently overwritten in the TFLite kernel?

Reply:
Yeah that's nicer, thanks :+1: